src/js/components/Anchor/__tests__/Anchor-test.js

```diff
@@ -188,4 +188,15 @@ describe('Anchor', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  test('weight renders', () => {
+    const component = renderer.create(
+      <Grommet>
+        <Anchor href="#" label="Normal" weight="normal" />
+        <Anchor href="#" label="Bold" weight="bold" />
```

Comment:
Can we add one more entry of test for using number on weight? Somethink like:
```suggestion
        <Anchor href="#" label="Bold" weight="bold" />
        <Anchor href="#" label="500" weight="500" />
```

Reply:
Sure. My only concern about that is that tests take into consideration proptypes, and passing `weight="500"` returns a `Warning: Failed prop type: Invalid prop 'weight' supplied to 'Anchor'.` which causes test to fail. So it looks like we can use `weight="500"` or `weight={500}` in our codes, but only the second one will work in this test.

Any suggestion? Should we just update test passing a number, or also add an example to storybook?

Reply:
You are right, my bad, change it to {500}. It shouldn't be a string.